src/main/java/com/conveyal/datatools/manager/models/FeedVersionSummary.java

```diff
@@ -11,14 +18,46 @@ public class FeedVersionSummary extends Model implements Serializable {
 
     public FeedRetrievalMethod retrievalMethod;
     public int version;
+    public String feedSourceId;
     public String name;
     public String namespace;
     public String originNamespace;
     public Long fileSize;
     public Date updated;
+    /** Only a subset of the validation results are serialized to JSON via getValidationSummary. */
+    @JsonIgnore
+    public ValidationResult validationResult;
```

Comment:
I reversed course for using raw `validationResult`, it has too much stuff imo, so with https://github.com/ibi-group/datatools-server/pull/464/commits/a6a72a7890847b13b4d7b604b09b04427e62c974, I am just exposing two fields from `FeedVersion.validationSummary` that are used in the UI (see https://github.com/ibi-group/datatools-ui/pull/792/commits/915e5afeacebe24b76934b2a3855715baa1bbbfb), any questions let me know @miles-grant-ibigroup.